### PR TITLE
feature/add-photo-to-reward

### PR DESCRIPTION
**Pull request by rniedziolka**
Hello,
The next feature is ready:
https://www.loom.com/share/0797dd61fdbf41ae8144ca339e4a6262
Look at this and give me your feedback!

### CODE REVIEW

**Comment by oskarlakner** [comment]
I miss specs here

**Review by oskarlakner — APPROVED** [review]

**Comment by rniedziolka** [comment]
Hey @oskarlakner,
Could you please take a look if that is ok?
https://github.com/rniedziolka/employee-recognition-platform-1637255604/pull/32/commits/50719765c5745223602581cb38b5f31a6cccaaf9
Thank you!

File: spec/system/admin_reward_spec.rb [hunk 1]
```diff
@@ -29,11 +30,22 @@
     fill_in 'Title', with: 'Example245'
     fill_in 'Description', with: 'Desc12390'
     fill_in 'Price', with: '260'
+    attach_file(File.absolute_path('./spec/files/images/default.jpg'))
     click_button 'Update Reward'
     expect(page).to have_content 'Example245'
     expect(page).to have_content '260.0'
+    expect(page).to have_selector("img[src$='default.jpg']")
```
Comment (ciejj) [inline spec/system/admin_reward_spec.rb:37]: I would rather use that (more or less):
`    expect(page).to have_css('img', text: "default.jpg")
`

**Review by ciejj — APPROVED** [review]
consider using `have_css` instead of `have_content`

File: spec/system/admin_reward_spec.rb (hunk 1 above)
Comment (rniedziolka) [reply spec/system/admin_reward_spec.rb:37]: ok, thanks, fixed

**Pull request merged**